src/ex1/style.css

```diff
@@ -34,39 +47,49 @@ img {
 }
 
 
+.list-item-taskDone-checkbox-pressed{
+    text-decoration: line-through;
+    padding: 7px;
+    border: 1px solid black;
+}
+
+.list-item-taskDone-checkbox-pressed:hover{
+    cursor: pointer;
+    background: linear-gradient(90deg, rgba(2, 0, 36, 1) 0%, rgba(255, 228, 196, 1) 0%, rgba(0, 212, 255, 1) 100%);
+}
 
-.taskDone-btn {
+
+.taskDone-checkbox-not-pressed {
     background-color: azure;
     float: right;
-    height: 2p0;
+    height: auto;
     width: auto;
 }
 
-.taskDone-btn-pressed {
+.taskDone-checkbox-pressed {
     text-decoration: line-through;
     padding: 7px;
     border: 1px solid black;
 }
 
-.container-addTodo {
+.container-addTask {
```

Comment:
naming convention for CSS classes is [kebab-case](https://www.theserverside.com/definition/Kebab-case)

so name this .container-add-task instead